### DISCUSSION TITLE
Supabase SQL Trigger Error: "record 'new' has no field 'image_url'"

### DISCUSSION THREAD

**Discussion by Matignaciom** [supabase/supabase#26248] [Questions] [answered]
When attempting to insert a new user in Supabase, the following error occurs:  

![image](https://github.com/supabase/supabase/assets/66745824/84f5cd6c-aef1-4df8-b0a0-8944731a000e)

![image](https://github.com/supabase/supabase/assets/66745824/ca2a1bfa-d435-4ad8-8670-8665666c7aac)

Any idea what is wrong?
A few days ago, this error did not occur, and I haven't made any modifications to my database. However, now this error is appearing.

**Comment by GaryAustin1** [comment, marked as answer]
Sort of confusing as the error is on `image_url` and not `imagen_url`.  Do you have another function involved, or did you,  using image_url?
Any more info in the Postgres logs?

**Reply by Matignaciom** [reply]
Hello! Sorry but I was able to solve it, the problem was that there was a duplicate trigger with a similar name (image_url), thank you!
